ArticleTemplates/assets/js/modules/youtube.js

```diff
@@ -254,8 +254,7 @@ define([
     }
 
     function onPlayerPlaying(id) {
-        var placeholderParent,
-            currentTime = Math.round(players[id].player.getCurrentTime());
+        var placeholderParent;
```

Comment:
This is an unrelated javascript validation fix that was flagged when I ran `yarn validate` locally.